lnst/Controller/Machine.py

```diff
@@ -280,6 +280,12 @@ def get_dev_by_hwaddr(self, hwaddr):
                 return dev
         return None
 
+    def get_dev_by_ifname(self, ifname):
+        for dev in list(self._device_database[self._initns].values()):
```

Comment:
what about other namespaces here? Since this is the `Machine` class it is used for all of the namespaces on the specific agent so wouldn't it be more sensible to have a more universal method?

Also, I see you've used this method in an older commit, ideally this would be switched - first a commit that adds a new method, then a commit that uses this new method.

Reply:
Fixed